manifests/web.pp

```diff
@@ -231,7 +231,7 @@
       # Check OS release for proper prefix
       case $facts['os']['name'] {
         'Ubuntu': {
-          if versioncmp($facts['os']['release']['major'], '16.04') >= 0 {
+          if versioncmp($facts['os']['release']['major'], '18.04') >= 0 {
```

Comment:
I think this whole condition can be dropped. You should never get to the `else` branch. Similar for Debian below. The default branch can't be reached so the whole case can be replaced with `$php_db_package = "php-${db}"`.

Reply:
cleaned it up